### ISSUE TITLE
Installing tidyverse package/s

### ISSUE THREAD

**Issue by jdrnevich** [carpentries-incubator/bioc-intro#73]
I'm prepping to teach using this workshop and I've made a few minor changes so far, but I've come across a bigger one I wanted to discuss before unilaterally making the change...

At the end of the R & RStudio episode, package installation and loading via `library()` are first discussed, and I felt that was OK when I went over it. But then we don't use any add on packages until the beginning of the https://carpentries-incubator.github.io/bioc-intro/30-dplyr/index.html lesson. The first code line says to `BiocManager::install("tidyverse")` but if they have done the set up they should already have tidyverse and this will unnecessarily do a re-install (we've had lots of problems with permissions & installing packages as more people use OneDrive). 

Of course, not everyone will have done the set up and will need to install. This got me to thinking that this could be a good place to cover the common `Error in library("kslknsknls") : there is no package called ‘kslknsknls’` that everyone runs into all the time. We could flip them and start with the `library(tidyverse)`, then have a little aside of "Did you get an error `Error in library("tidyverse") : there is no package called ‘tidyverse’`? This means you have not installed the package yet. To install the package, do ...

What do you think?

**Comment by jdrnevich** [comment]
This would probably also satisfy open issue #16

**Comment by lgatto** [comment]
Yes, it would be great to have a 'common issues' paragraph in the setup section.
